server/src/modules/persisters/profile-elements-persisters.ts

```
@@ -0,0 +1,187 @@
+import { Context } from '../../config/context';
+import { BaseModel } from '../../database/base-model';
+import DatabaseProfileElements, { DatabaseProfileElementsInitializer } from '../../types/database/DatabaseProfileElements';
+
+export interface ProfileElementsModel extends DatabaseProfileElements {}
+
+export class ProfileElementsModel extends BaseModel {
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)